playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json

```diff
@@ -13,7 +13,7 @@
         "relatesTo": [
           "$.currentAuthenticatorEnrollment"
         ],
-        "name": "challenge-authenticator", 
+        "name": "challenge-poll",
```

Comment:
Don't you need to update https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/RemediationConstants.js#L21?

Reply:
I also see that there are some more API mock files still with name `challenge-authenticator`. Can you update them all?

Reply:
@jhe-okta  did I miss any fastpass related mock that I should have updated? This change is not to completely remove 'challenge-authenticator" as it is relevant in a few cases. eg) totp.

All tests are passing and I am not sure what I am missing here.

Reply:
I think all the mock files start with `identify-` and contain `challenge-authenticator`.

Reply:
@jhe-okta  I double checked the only files relevant to OV fastpass that has 'challenge-authenticator' have been fixed 

https://github.com/okta/okta-signin-widget/search?l=JSON&p=1&q=challenge-authenticator